scripts/Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM alpine
 MAINTAINER chriseth <chris@ethereum.org>
 
+#Establish working directory as soliditiy
+WORKDIR /solidity
+#Determine whether or not we are in the testing suite
+ARG TEST=Off
+#Copy working directory on travis to the image
+COPY / $WORKDIR
+
+#Install dependencies, eliminate annoying warnings, test if we are in a test environment, if so, 
+#add in soltest to the binary. If not, assume this is basic solidity image, build it and ship it out.
 RUN \
-  apk --no-cache --update add build-base cmake boost-dev git                                                && \
-  sed -i -E -e 's/include <sys\/poll.h>/include <poll.h>/' /usr/include/boost/asio/detail/socket_types.hpp  && \
-  git clone --depth 1 --recursive -b develop https://github.com/ethereum/solidity                           && \
-  cd /solidity && cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSTATIC_LINKING=1                             && \
-  cd /solidity && make solc && install -s  solc/solc /usr/bin                                               && \
-  cd / && rm -rf solidity                                                                                   && \
-  apk del sed build-base git make cmake gcc g++ musl-dev curl-dev boost-dev                                 && \
+  ./scripts/install_deps.sh                                           									    										&& \
+  sed -i -E -e 's/include <sys\/poll.h>/include <poll.h>/' /usr/include/boost/asio/detail/socket_types.hpp  										&& \
```

Comment:
Not to make it run correctly. Moreso to rid irritating warnings. Its been the case for some time and is the case with the current alpine docker file.